src/swig/CMakeLists.txt

```diff
@@ -71,3 +77,43 @@ if (GUILE_FOUND)
   set (GUILE_EXTENSION_PATH ${_REL_GUILE_EXTENSION_PATH})
   install (TARGETS nlopt_guile LIBRARY DESTINATION ${GUILE_EXTENSION_PATH})
 endif ()
+
```

Comment:
the guile install source must be changed too:
```
install (FILES ${CMAKE_CURRENT_BINARY_DIR}/nlopt.scm DESTINATION ${GUILE_SITE_PATH})
```

Reply:
Should be fixed in the latest revision (commit 4ad857746f2e1b69c2fd610f3e6ecafa6356397c).